draft-ietf-rats-architecture.md

```diff
@@ -273,12 +273,24 @@ plans to support in the TEEP architecture {{?I-D.ietf-teep-architecture}}.
 
 # Two Types of Environments
 
-An Attester consists of at least one Attesting Environment and Attested Environment.
-In some implementations, the Attesting and Attested Environments might be combined.
-Other implementations might have multiple Attesting and Attested Environments.
-
-    <this section can include Two Types of Environments content from draft-birkholz-rats-architecture
-    but can we find a better name? also this could be a subsection of something else?>
+An Attester consists of at least one Attesting Environment and at least one
+Target Environment. In some implementations, the Attesting and Target Environments
+might be combined.
+Other implementations might have multiple Attesting and Target Environments.
+One example is a set of components in a boot sequence (e.g., ROM, firmware,
+OS, and application) where a Target Environment is the
+Attesting Environment for the next environment in the boot sequence.
```

Comment:
The carrier-grade router, one kind of composite device, is consisted of several slots, and each slot has this set of components in a boot sequence.
Calling the router "attester" and the slot "attesting environment" isn't appropriate, I think. Maybe call the router "Attester" and the slot "sub-attester"?

Reply:
Using "Attester" and "Sub-Attester" may give a better understanding to explain the composite device is composing of what.

Reply:
I think this is the "layered attestation" concept or "change of roles during the remote attestation procedure" that we should (at least I think so) dedicate another section to.

Reply:
Eric has volunteered to write layered attestation text.